examples/contrib/tracking/plot_utils.py

```diff
@@ -0,0 +1,71 @@
+import torch
+from matplotlib import pyplot
+import time
+
+
+def init_visdom(visdom_flag):
+    if visdom_flag:
+        try:
+            from visdom import Visdom
+            viz = Visdom()
+            startup_sec = 1
+            while not viz.check_connection() and startup_sec > 0:
+                time.sleep(0.1)
+                startup_sec -= 0.1
+            assert viz.check_connection(), 'No connection could be formed quickly'
```

Comment:
nit: I think this would be simpler as
```py
if viz.check_connection():
    return viz
else:
    warnings.warn('No connection could be formed quickly')
    return None
```

Reply:
I wanted to catch all the `*Error`s urllib throws if visdom is not running but I can't catch them no matter what I do 😄

Reply:
I'm not sure why you're catching an `AssertionError`. Maybe you could broaden to `Exeption`
```py
try:
    ...
except Exception as e:
    warnings.warn('Failed to create Visdom instance: {}'.format(e))
    return None
```